### DISCUSSION TITLE
Font color

### DISCUSSION THREAD

**Discussion by Arczer911** [darkreader/darkreader#7940] [Q&A]
Hi guys,

Could you tell me pls how to change pink and blue font color ?
Please see the attachment

![screenshot-www google pl-2022 01 13-00_25_44](https://user-images.githubusercontent.com/97641538/149239283-299eecac-4838-47dd-80c5-95796f08fcc9.png)

**Comment by skoufaki** [comment]
I would generally like to change the font color of a website's text.

**Comment by Gusted** [comment]
It's already implemented: https://github.com/darkreader/darkreader/issues/650

**Reply by Arczer911** [reply]
> It's already implemented: #650

Thank you for your response, 
However I can't find the option that is responsible for changing these colors.

On top of that I would like to change a color of bold font in google search results. 
It's not visible enough.
I'm talking about "dark reader", "pink" etc. in the sampel above.

Can you please tell me how to do it ?

**Reply by Gusted** [reply]
> However I can't find the option that is responsible for changing these colors.

See: https://github.com/darkreader/darkreader/issues/650#issuecomment-733599758



> However I can't find the option that is responsible for changing these colors.
> 
> On top of that I would like to change a color of bold font in google search results. It's not visible enough. I'm talking about "dark reader", "pink" etc. in the sampel above.

Due to the way dark reader works, when we create the colors we don't know to which elements they will be applied to, so it's not possible to change the color of only bold elements.
